src/Core/Settings.h

```diff
@@ -842,6 +842,9 @@ class IColumn;
     M(Bool, print_pretty_type_names, false, "Print pretty type names in DESCRIBE query and toTypeName() function", 0) \
     M(Bool, create_table_empty_primary_key_by_default, false, "Allow to create *MergeTree tables with empty primary key when ORDER BY and PRIMARY KEY not specified", 0) \
     M(Bool, allow_named_collection_override_by_default, true, "Allow named collections' fields override by default.", 0)\
+    M(String, default_view_sql_security, "DEFINER", "Allows to set a default value for SQL SECURITY option when creating view.", 0) \
+    M(String, default_view_definer, "CURRENT_USER", "Allows to set a default value for DEFINER option when creating view.", 0) \
+    M(Bool, stop_insert_if_fail_to_update_dependent_view, true, "If true, insert queries will fail if any of the underlying materialized view inserts have failed.", 0) \
```

Comment:
This is a behaviour change for ClickHouse. It at least should be represented in SettingChangesHistory and I recommend to disable it by default. But overall why do we care about it in this PR? If some INSERTs to Materialized Views failed, because a DEFINER user doesn't have grants to push data - this is Ok. At least it is backward compatible. Currently customers would have the same problem for example when they add another view to the chain and the database user they are using doesn't have a required GRANT.

Reply:
Okay, I was told that there is no change in behaviour.